update VST installation path

ildyria opened the pull request:
installation path (was already updated in the release part)

This should allow user to be able to directly use VST via opam.

gares commented:
Travis found two problems:
- the `opam` file contains some empty fields
- the installation fails

Could you please check on these issues?

ildyria commented:
I updated the maintainer field and a bunch of other parameters.
However the installation failure is not of my domain: 

```
=-=- Error report -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
The following actions failed
  ∗  install coq-compcert dev
The following changes have been performed
  ∗  install coq        dev     
  ∗  install menhir     20171222
  ∗  install num        1.1     
  ∗  install ocamlbuild 0.12.0  
```

lastland commented:
What's the status of this PR?

liyishuai commented on extra-dev/packages/coq-vst/coq-vst.dev/opam:
```diff
@@ -1,22 +1,36 @@
+	]
 depends: [
-  "coq" {= "dev"}
+  "coq" {>= "8.8.0"}
```
Do we need such late versions of Coq and CompCert?

ildyria replied:
@liyishuai Indeed, we should allow  `>= 8.7.0`.
Will update that.

@lastland At the moment, this PR is useless, this pull request will be usefull as soon as this other one  ( https://github.com/AbsInt/CompCert/pull/228 ) is accepted.
**CompCert does not install any `.vo` files at the moment.** As such the opam package of CompCert is installing the compiler binaries but is completely useless for any coq developer as the `.vo` files are not accessible. :disappointed:

ildyria commented:
@lastland At the moment, this PR is useless, this pull request will be usefull as soon as this other one  ( https://github.com/AbsInt/CompCert/pull/228 ) is accepted.
**CompCert does not install any `.vo` files at the moment.** As such the opam package of CompCert is installing the compiler binaries but is completely useless for any coq developer as the `.vo` files are not accessible. :disappointed:

gares commented:
What should we do with this? Still relevant?

lastland commented:
@gares This PR depends on a change to CompCert (https://github.com/AbsInt/CompCert/pull/228). It has not been accepted by CompCert yet. I just made a follow-up question to the CompCert people about what they think (https://github.com/AbsInt/CompCert/issues/227). Maybe we can wait for their response and decide what to do with this PR next?

ildyria commented:
Rebased and updated coq-compcert options so they match PR https://github.com/AbsInt/CompCert/pull/232 

The path to install the `.vo` files is updated in such way that 
```
To use, pass the following to coq_makefile or add the following to _CoqProject:"; echo "-R $(COQDEVDIR) compcert
```
is not required.

gares commented:
CI says that `ia32` is not a supported target. Our CI uses 64 bit OCaml if that matters. Any idea?

ildyria commented:
Now that https://github.com/AbsInt/CompCert updated their `configure.sh`, it should provide a different result.

ildyria commented:
Build fails because 
```
Makefile:42: *** COMPCERT_VERSION=version=3.2 buildnr= tag= but /home/travis/opam-root/4.06.0/lib/coq/user-contrib/compcert/VERSION=.  Stop.
```

gares commented:
Is this PR still actual? I've merged the other one. This one, on ocaml 4.06, seems to fail early

ildyria commented:
@gares This should solve all the previous problems.

And this PR is still live if someone wants to work with the master trunk of VST instead of a tag version. :)
